blocks/algolia-assortment-content-source-block/README.md

```diff
@@ -0,0 +1,26 @@
+# @wpmedia/algolia-assortment-content-source-block
+
+The Algolia assortment content source block allows you to display products from an Algolia index. See Algolia docs for more content info: https://www.algolia.com/doc/guides/sending-and-managing-data/prepare-your-data/#algolia-records
+
+## ANS Schema
+
+not applicable
+
+## Configurable Params
+
+```
+    filters: "text",
```

Comment:
Wondering if we could expand on the params a bit here?  More description and examples possibly?

Reply:
updated with some description and links to algolia's documentation. I don't want to recreate their docs too much

Reply:
Updated! thanks for the feedback @visionbegin. I added some links to algolia's documentation related. I don't want to recreate their docs too much (they have really great docs with examples).